.scss-lint.yml

```diff
@@ -83,6 +85,9 @@ linters:
     NestingDepth:
         max_depth: 5
 
+    PseudoElement:
+        enabled: false
```

Comment:
This rule enforces writing pseudo elements with the correct syntax, `.class::before`. It's not a big deal, we'll find a tool that reformats all of the files at once when we get to this.